docs/tutorials/benchmarking/bm_local.md

```diff
@@ -0,0 +1,148 @@
+# Benchmarking with Local Back-end
+
+The local back-end is the default choice in Syne Tune for running on real
+benchmarks (i.e., not based on tabulated data, but on real training scripts).
```

Comment:
first time users read about real benchmarks. lets define it prior to this script.

Reply:
OK